### DISCUSSION TITLE
$pageName for search, for page with multiple tables

### DISCUSSION THREAD

**Discussion by eswachsman** [rappasoft/laravel-livewire-tables#311] [Q&A]
When using multiple table on a page I can use the `$pageName` property to change the url parameter for pagination. Is there a way to do the same for search?

Meaning, I want to have a link on another page, linking to this page and one table should have search prefilled. I currently use `<a href="/mypage?filters%5Bsearch%5D=NAVY">` but both tables on that page get searched.

https://github.com/rappasoft/laravel-livewire-tables/wiki/Multiple-tables-on-a-single-page

**Comment by lrljoe** [comment]
This is present in v2+
